e2e/registerENSFlow.spec.js

```diff
@@ -232,10 +232,7 @@ describe('Register ENS Flow', () => {
     await Helpers.delay(60000);
   });
 
-  it('Should see confirm registration screen and set reverse records', async () => {
-    await Helpers.checkIfVisible(`ens-reverse-record-switch`);
-    // set RANDOM_NAME as primary name
-    await Helpers.waitAndTap('ens-reverse-record-switch');
```

Comment:
not following why this step was removed, this is checking the switch is off since the rainbowtestwallet.eth has an ens, so the e2e is turning it on to send that tx